virttest/remote.py

```diff
@@ -211,6 +213,20 @@ def handle_prompts(session, username, password, prompt, timeout=10,
             elif match == 13:  # console prompt
                 logging.debug("Got console prompt, send return to show login")
                 session.sendline()
+            elif match == 14:  # VMware vCenter command prompt
+                logging.debug(
+                    "Got VMware VCenter prompt, send 'shell' to launch bash")
+                if not shell_enabled:
```

Comment:
shell_enabled is hard to 'false' by you, so whatever the execution will enter here. Maybe just remove the judgement,and just run 'shell.set --enabled True ' and 'shell' squencially.

Reply:
Done